### PR TITLE
 add enter key press submission, filter messages based on user, styling tweaks

### PR DESCRIPTION
**Pull request by esayler**
@dylanavery720 Hey Dylan, made enter key work to submit chat messages and hid the user list on mobile. Also changed the color of the logout button.

### CODE REVIEW

File: lib/components/MessageWindow.jsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ const MessageWindow = ({ messages, handleDelete, user }) => {
   return (
     <section className="message-window" >
      {messages.map(m => <MessageCard
+       key={m.key}
```
Comment (dylanavery720) [inline lib/components/MessageWindow.jsx:8]: good to see those stupid warnings go away!

**Review by dylanavery720 — APPROVED** [review]
Right on. Lets try to write some tests in the morning and maybe make some Mixins. Meexeens.

Dylan

File: lib/components/MessageWindow.jsx (hunk 1 above)
Comment (esayler) [reply lib/components/MessageWindow.jsx:8]: Yeah dem warnings were on point!

**Comment by esayler** [comment]
Nice job on cleaning up the sass and cool tests so far. Good work! 💯

**Pull request merged**